data/authors/razvan.toml

```diff
@@ -0,0 +1,13 @@
+name = "Răzvan C. Rădulescu"
+user = "razvan"
+picture = "img/avatar/razvan.png"
+description = "GameDev. enthusiast, passionate about technology, sciences and philosophy. I am an avid adept of spreading the knwledge."
```

Comment:
Typo on knowledge. Also, how about `I am all about sharing knowledge.` or `I believe we should freely share our knowledge.`, or something like that?

Reply:
Yep, I rephrased